src/settings.rs

```diff
@@ -81,6 +81,7 @@ pub struct AI {
     pub limit_reset_duration_in_sec: i64,
     #[serde_as(as = "Base64")]
     pub explain_sign_key: [u8; 32],
+    pub history_deletion_period_in_sec: u64,
```

Comment:
I wonder if it's a conscious choice that we don't define a default value here (same applies to `limit_reset_duration_in_sec`), to avoid having to specify it in our deployment?

Reply:
That would need a bit of refactoring in the configuration machinery that is out of scope here. I agree that it would be nice to have sane defaults for most of the configuration.